### PR TITLE
Solved test book and category

### PR DESCRIPTION
**Pull request by Krasnov-Maksim**

### CODE REVIEW

File: pom.xml
Comment (koval-kn) [inline pom.xml:1]: not implemented:
Please add a screenshot with the test coverage result into the PR description

File: src/test/java/mate/academy/bookstore/controller/BookControllerIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+import org.springframework.web.context.WebApplicationContext;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+class BookControllerIntegrationTest {
+    private static final int PAGE_NUMBER = 0;
+    private static final int PAGE_SIZE = 10;
+    private static MockMvc mockMvc;
+    @Autowired
+    private ObjectMapper objectMapper;
```
Comment (koval-kn) [inline src/test/java/mate/academy/bookstore/controller/BookControllerIntegrationTest.java:59]: jsonpath can be used instead to check response values:
`.andExpect(jsonPath("$.id").value(1))`
```suggestion
```
fix this in the whole project

File: pom.xml
Comment (koval-kn) [inline pom.xml:1]: why your PR contains changes not related with tests?

**Review by koval-kn — CHANGES_REQUESTED** [review]

**Comment by Krasnov-Maksim** [comment]
Capture

**Review by koval-kn — APPROVED** [review]

**Pull request merged**